corehq/apps/export/static/export/js/list_exports.ng.js

```diff
@@ -80,6 +80,11 @@
             var currentUrl = useLegacyBulkExportUrl ? $scope.legacy_bulk_download_url : $scope.bulk_download_url;
             input.closest("form").attr("action", currentUrl);
         };
+        $scope.downloadRequested = function($event) {
+            var $btn = $($event.target);
+            $btn.addClass('disabled');
```

Comment:
Aha. I hadn't heard of `pointer-events` (though it looks like it's been around for a long time). Nice.